committeeoversightapp/models.py

```diff
@@ -23,6 +24,25 @@
                                              EventDocument
 
 
+class HearingEvent(Event):
+    class Meta:
+        proxy = True
+
+    @property
+    def category(self):
+        try:
+            return HearingCategory.objects.get(
+                event_id=self.id
+            ).category
+        except ObjectDoesNotExist:
```

Comment:
100% fine to import this exception for use as you've done here, just want to share some related trivia: You can access a `DoesNotExist` exception via any Django model, too, e.g., `HearingCategory.DoesNotExist`. Accomplishes the same thing, just minimally more convenient. 🙃